phpBB/phpbb/notification/method/email.php

```diff
@@ -68,6 +98,53 @@ public function is_available(type_interface $notification_type = null)
 		return parent::is_available($notification_type) && $this->config['email_enable'] && $this->user->data['user_email'];
 	}
 
+	/**
+	* {@inheritdoc}
+	*/
+	public function get_notified_users($notification_type_id, array $options, $notification_type_name)
+	{
+		$notified_users = array();
+
+		if ($notification_type_name == 'notification.type.post' && !empty($options['item_parent_id']))
```

Comment:
you don't need the name: you already have the notification ID and you should use that to retrieve the name if you need it

Reply:
I didn't want to add yet another query to the process since the data are already available.

Reply:
it won't add more queries, we have helpers for that which are cachin the results (and the cache will already be loaded here)

Reply:
Can you direct me to the method that converts ID to name? Because in the manager, we only have a name-to-id method, not the other way around...